templates/takeovers/_robotics-takeover.html

```diff
@@ -0,0 +1,25 @@
+<section class="p-strip--image js-takeover is-dark p-takeover--robotics">
+  <div class="row u-vertically-center">
+    <div class="col-8 u-fade-left--medium">
+      <h1>適切なロボットOSの選択</h1>
+      <p class="u-no-padding--top p-heading--four">開発・本稼働における要点の分析</p>
+      <a href="/engage/robotics_whitepaper?utm_source=Takeover&utm_medium=Takeover&utm_campaign=2)CY19_IOT_Robotics_Whitepaper_Japan" class="p-button--positive">ホワイトペーパーをダウンロードする</a>
```

Comment:
2)CY19_IOT_Robotics_Whitepaper_Japan is that correct?

Reply:
Yep, it's in the [brief](https://docs.google.com/spreadsheets/d/1-t6Lz_1sMnToPL8bdd_7alz9CTZyWlqcZ8E8YJq2BAo/edit#gid=127581435). I did wonder if the parenthesis was an unsafe character, but it seems like it's safe to use if it's part of a query string, at least according to [this](https://stackoverflow.com/questions/13225028/is-it-ok-to-have-brackets-or-parenthesis-in-url).